### ISSUE TITLE
Item Birthday does not reset when new character in same world

### ISSUE THREAD

**Issue by aawilson** [CleverRaven/Cataclysm-DDA#11700]
I have a tanning pelt in a save that has been "tanning" through several iterations of characters in the same world. My suspicion is that the pelt birthday is not reset with the new character, meaning it won't return true on `time_to_do` here: https://github.com/CleverRaven/Cataclysm-DDA/blob/b979c6446753931c7dc0738a1651c3db1872673b/src/iuse_actor.cpp#L410, but I don't know how to inspect the item directly to confirm my suspicion.

**Comment by BevapDin** [comment]
That is a common problem when several characters play in the same world. If the birthday would be "reset" (to what exactly?), the other character (which does not need to be alive) could use it directly.

Example:
- Start with character A, player for a (in game) month, make a tanning hide (and drop it somewhere).
- Save the game (character A is still alive) and start a new game with character B.
- Go to the tanning hide with character B (still on the first day), according to your suggesting, the birthday of the hide would be reset, but to what exactly?
- Save the game again and reload character A, which can now immediately use the tanning hide (its birthday was reset).

Bottom line:
One doesn't know to what to reset those ages to and when to do so.

**Comment by mugling** [comment]
No obvious path to resolution:

> One doesn't know to what to reset those ages to and when to do so
